webextension/scripts/utils.js

```diff
@@ -116,7 +116,7 @@ if (typeof isInTestEnv === 'undefined') {
  */
 function isBadgeOnTop() {
   // TODO: check other browsers
-  const badgeOnTop = { firefox: true, safari: true, chrome: false }
+  const badgeOnTop = { firefox: true, safari: true, chrome: false, edge: false }
```

Comment:
I suggest to ensure that a default ia returned if the browser key is not mentioned.

Reply:
It's intentional to return `undefined` with unknown browsers. Works like false when in an if expression. Means we don't know what the correct value is yet.

Reply:
As long as the extension doesn't crash and shows some icon, even if not the best one, that's fine.

Reply:
Of course! JS ain't Java or C! And I tested this, it works.